optuna/storages/_redis.py

```diff
@@ -463,19 +465,19 @@ def set_trial_param(
 
         trial = self.get_trial(trial_id)
 
-        with self._redis.pipeline() as pipe:
-            pipe.multi()
-            # Set study param distribution.
-            param_distribution[param_name] = distribution
-            pipe.set(
-                self._key_study_param_distribution(study_id), pickle.dumps(param_distribution)
-            )
+        queries: Mapping[Union[str, bytes], Union[bytes, float, int, str]]
```

Comment:
I have questions about type hinting.

1. Can the type of the keys be `str`? Both `_key_study_param_distribution` and `_key_trial` seems to return `str`.
2. Can the type of the values be `bytes`? The [pickle reference](https://docs.python.org/3/library/pickle.html#pickle.dumps) says that `pickle.dumps` returns a bytes object.

Reply:
Thank you four your review! Indeed we only dealing with the types, but when we define like `queries: Mapping[str, bytes]`, `mypy` complains to it...

```
optuna/storages/_redis.py:480: error: Argument 1 to "mset" of "BasicKeyCommands" has incompatible type "Dict[str, bytes]"; expected "Mapping[Union[str, bytes], Union[bytes, float, int, str]]"
Found 1 error in 1 file (checked 275 source files)
```

Reply:
@toshihikoyanase Is there any comment ?

Reply:
Thanks !